newdle/schemas.py

```diff
@@ -44,29 +53,47 @@ def sort_users(self, data, many, **kwargs):
 
 
 class NewUnknownParticipantSchema(mm.Schema):
+    id = fields.Integer()
```

Comment:
This only makes sense when editing, but not when *creating* a new unknown participant (in that case it's inserting a new row, so letting the user pass the ID would be problematic). Especially since we pass `**args` to the model in `create_unknown_participant`, so it would actually allow you to set a custom ID.

Reply:
Good point, I was undecided on whether to remove it or exclude it (`exclude=('id',)`) for the particular case of `create_unknown_participant`. I eventually think it makes more sense to exclude it since, by definition, an unknown participant is non-user but may or not be registered already.

Reply:
Wouldn't it be more logical to have an `UnknownParticipantSchema` that contains the ID, and a `NewUnknownParticipantSchema` (superclass?) that doesn't? Sure, the `exclude` in the API function may work (I guess you tested it?), but having this done on the schema level feels safer and cleaner...

Reply:
Yes, I just wanted to avoid the cardinality of participant schemas from growing, as we already have: `NewUnknownParticipantSchema`, `NewKnownParticipantSchema`, `NewParticipantSchema` (plus eventually `UnknownParticipantSchema` or `KnownParticipantSchema`). But apart from that, it feels safer for me too.